go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/philhofer/fwd v1.1.3-0.20240612014219-fbbf4953d986 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/valyala/tcplisten v1.0.0 // indirect
+	golang.org/dl v0.0.0-20241001165935-bedb0f791d00 // indirect
```

Comment:
Why is this here?

Reply:
This is an official multi-version control function of the go language, because the Go version I use locally is higher than the go version in fiber's go mod. In order to keep the version consistent, I rely on this method to use it locally in the same way as the fiber framework. Go language version, you can see this to get more information: https://pkg.go.dev/golang.org/dl and https://pkg.go.dev/golang.org/dl@v0.0.0-20241001165935-bedb0f791d00/go1.22.6 . It will not have any impact on the function of the fiber